src/Components/HabitTracker/HabitItem.js

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react';
+
+const TrackerButton = ({ title, duration }) => {
+  // Creating an array that will handle all the <div> and then we will display only this "progress" array
+  const progress = [];
+  for (let i = 1; i <= duration - 1; i++) {
+    //the first push is for the rounded circle with the number in it
+    progress.push(
+      <div className="flex-1" key={i}>
+        <div
+          id={i}
+          onClick={() => {
+            if (
+              document.getElementById(i).className ===
+              'bg-lightGrey w-4 h-4  mx-auto rounded-full text-lg text-white flex items-center'
```

Comment:
try not to write those classes many times...
u can either save them in a variable or something else